packages/modules/web_themes/colors/source/index.html

```diff
@@ -22,6 +22,7 @@
 	<link rel="icon" type="image/png" sizes="16x16" href="/openWB/web/img/favicons/favicon-16x16.png">
 	<link rel="apple-touch-icon" sizes="57x57" href="/openWB/web/img/favicons/apple-touch-icon-57x57.png">
 	<link rel="apple-touch-icon" sizes="60x60" href="/openWB/web/img/favicons/apple-touch-icon-60x60.png">
+	<link rel="manifest" href="manifest.json">
```

Comment:
Bitte keine eigene Datei `manifest.json` anlegen, sondern wie bei den Fav-Icons auf das openWB Manifest verlinken.
```suggestion
	<link rel="manifest" href="/openWB/web/manifest.json">
```

Reply:
Done